### ISSUE TITLE
Breaks speaker phone on 3rd party applications

### ISSUE THREAD

**Issue by AdamMajer** [vector-im/element-android#4844]
### Steps to reproduce

1. I use SIP voicecalls (Zoiper) . Enable speaker phone
2. Elements received message
3. speaker phone is turned off

or if I switch to elements while on speaker phone, the speaker phone is turned off. Same things happens during regular phone calls.

It affects my Nokia 7.1 with Android 10 or an old phone from 5 years ago with older Android.

### Outcome

Do not adjust sound devices when application becomes active.

### Will you send logs?

No

**Comment by ouchadam** [comment]
to double check, does this happen with any other app, such as receiving an sms/phone call for example?

**Comment by AdamMajer** [comment]
No, it only happens with Element Android. It does not happen with any other app.

Somehow the notification sound resets the phone from speaker mode to normal. I saw previously that people were complaining that notifications were sent on speaker and that was "fixed". I wonder if those fixes were related to this bug.
